src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj

```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<Description>Microsoft Azure Monitor Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Monitor</AssemblyName>
-		<Version>0.19.0-preview</Version>
+		<Version>0.19.1-preview</Version>
```

Comment:
Looks like a breaking change, please bump this to `0.20.0-preview`

Reply:
This is a bug fix (issue #2655 from Swagger): any call that used the top argument returned BadRequest.